### ISSUE TITLE
Entering and exiting comment frame not properly announced in Word 2016

### ISSUE THREAD

**Issue by leonardder** [nvaccess/nvda#7003]
STR:
1. Open a new document in MS Word
2. Press ctrl+alt+m to create a comment

* Expected: NVDA reports: 'comments frame'
* actual; nvda reports nothing at all. Only after a down arrow press, NVDA reports 'Comments frame'

Same happens for closing the frame with escape. NVDA doesn't announce its closing, only after moving the caret.

**Comment by feerrenrut** [comment]
I can reproduce this with NVDA Version `next-13949,b9c7808e`. It would make it a lot clearer to know that you are in a comments frame with the suggested expected behaviour. I'm not aware of any technical limitations on this. Perhaps @michaelDCurran can comment on that?

**Comment by michaelDCurran** [comment]
That specific key command will need a script, which after sending the 
key through, fetches and speaks the line at the caret. This will cause 
the frame change to be announced as part of the textual content for the 
line.

**Comment by Qchristensen** [comment]
Just a comment that this issue is still present with the current Office 365 (version 16.0.11029.20079 ) and NVDA 2018.3.2 and 2018.4rc1.

**Comment by Qchristensen** [comment]
Issue still present in NVDA 2019.1.1.  Just edited issue title to expand scope from Office 2016 to 2016 and 365
